src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -86,7 +90,39 @@ protected Task<ImmutableArray<CodeActionOperation>> GetPreviewOperationsAsync(Ca
                 async () => await CodeAction.GetPreviewOperationsAsync(cancellationToken).ConfigureAwait(false), cancellationToken);
         }
 
-        public virtual void Invoke(CancellationToken cancellationToken)
+        public void Invoke(CancellationToken cancellationToken)
```

Comment:
This terrifies me. My suggestion is we fix the editor, either by the editor team fixing this, or us deeming it important enough for Update 2 to fix it ourselves in their codebase.